src/main/java/core/basesyntax/dto/FruitRecordDto.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.dto;
+
+import core.basesyntax.model.Fruit;
+
+public class FruitRecordDto {
+    private Fruit fruit;
+    private Type type;
```

Comment:
wouldn't it be more logical if your `FruitRecordDto` contained `Fruit (its name)`, `Type` and `amount`?

and `Fruit` contained only `name` field?
your storage may look like map with `Fruit` as key and amount as value